charts/elastic-agent/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
I know this a default template. from `helm init`. We should remove it or refactor it to perform a meanful sanity check on elastic agent.